openvpn.php

```diff
@@ -0,0 +1,19 @@
+<?php
+require_once('models/ConnectionManager.php');
+
+list($script, $action, $user) = $argv;
+// $argv[0] is script location
```

Comment:
Not sure if the location of the script is needed since the BASH script will actually handle that part; `$argv[0]` should be the location of the tmp file that OpenVPN generates